docs/settings.rst

```diff
@@ -75,6 +75,10 @@ tokens submitted to the refresh view will be added to the blacklist.
 When set to ``True``, causes refresh tokens submitted to the
 ``TokenRefreshView`` to be added to the blacklist if the blacklist app is in
 use and the ``ROTATE_REFRESH_TOKENS`` setting is set to ``True``.
+You need to add ``'rest_framework_simplejwt.token_blacklist',`` to your 
+``INSTALLED_APPS`` in the django settings file to use this settings.
+
+Learn more about :doc:`/blacklist_app`.
```

Comment:
Shouldn't this be a :ref:, not a link? (Not good with docs). Otherwise, I think it's ok to add this. Just capitalize "Django" or remove the word entirely to be in accordance with DSF.

Reply:
> Just capitalize "Django" or remove the word entirely to be in accordance with DSF.

Sure, I think removing it would be better. So, removed it. :)


It looks like this. ref: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-doc
![Screenshot from 2020-09-25 21-29-49](https://user-images.githubusercontent.com/24854406/94286048-4e1d7a80-ff76-11ea-8525-57dcd1329268.png)

Maybe enabling Read the Docs pull request builder would be great for reviewing docs.:)

Reply:
Hey @saadmk11 Never heard of it, but if you'd like to help incorporate that to, I'd love to see it! Would make my life easier :)

Reply:
@Andrew-Chen-Wang  Sure! :+1:  

It's fairly simple you just need to enable it from Read the Docs dashboard.
You need to go to ``Admin > Advanced settings`` for this project on Read the Docs dashboard and Enable the ``Build pull requests for this project`` option.

More details here: https://docs.readthedocs.io/en/stable/guides/autobuild-docs-for-pull-requests.html

Would love to contribute to this project, I'm new to this codebase but I'll look through some issues. :)

Reply:
Awesome! I may be a maintainer, but I don't have access to bumping versions, PyPi, and RTD so if there is anything that we do need to enable externally from GitHub, please let me know as only David has those aforementioned capabilities!

Reply:
Nope, you need to enable the option that I mentioned from the RTD dashboard.

Reply:
Again, I don't have that capability. It was just a future reference in that you'll need to tell me detailed of what to enable.